### PR TITLE
Simple Animations

### PR DESCRIPTION
**Pull request by nicolasmederoc**
### Summary
 - Added a animation with various objects.
 - Added an Alert Dialog with simple instructions.

### Screenshot
MotionLayout alert dialog

MotionLayout

### CODE REVIEW

File: app/src/main/res/layout/fragment_motion_layout_animations.xml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:id="@+id/vMotionLayoutAnimations"
+    app:layoutDescription="@xml/animations_scene"
+    tools:showPaths="true">
+
+    <TextView
+        android:id="@+id/square"
+        android:layout_width="64dp"
+        android:layout_height="64dp"
```
Comment (lfmontesanto) [inline app/src/main/res/layout/fragment_motion_layout_animations.xml]: dimens

File: app/src/main/res/layout/fragment_motion_layout_animations.xml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        android:gravity="center"/>
+
+    <TextView
+        android:id="@+id/title_position"
+        android:layout_width="64dp"
+        android:layout_height="64dp"
```
Comment (lfmontesanto) [inline app/src/main/res/layout/fragment_motion_layout_animations.xml]: same

File: app/src/main/res/layout/fragment_motion_layout_animations.xml [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+        android:gravity="center"/>
+
+    <ImageView
+        android:id="@+id/plane_mode_image"
+        android:layout_width="40dp"
+        android:layout_height="40dp"
```
Comment (lfmontesanto) [inline app/src/main/res/layout/fragment_motion_layout_animations.xml]: same

File: app/src/main/res/layout/fragment_motion_layout_menu.xml [hunk 4]
```diff
@@ -8,13 +8,13 @@
 
 
     <Button
-        android:id="@+id/button2"
+        android:id="@+id/bAnimationsMotion"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
         android:layout_marginStart="8dp"
         android:layout_marginTop="8dp"
```
Comment (lfmontesanto) [inline app/src/main/res/layout/fragment_motion_layout_menu.xml]: same

File: app/src/main/res/layout/fragment_motion_layout_animations.xml (hunk 1 above)
Comment (nicolasmederoc) [reply app/src/main/res/layout/fragment_motion_layout_animations.xml]: done

File: app/src/main/res/layout/fragment_motion_layout_animations.xml (hunk 2 above)
Comment (nicolasmederoc) [reply app/src/main/res/layout/fragment_motion_layout_animations.xml]: done

File: app/src/main/res/layout/fragment_motion_layout_animations.xml (hunk 3 above)
Comment (nicolasmederoc) [reply app/src/main/res/layout/fragment_motion_layout_animations.xml]: done

File: app/src/main/res/layout/fragment_motion_layout_menu.xml (hunk 4 above)
Comment (nicolasmederoc) [reply app/src/main/res/layout/fragment_motion_layout_menu.xml]: d o n e

**Pull request merged**